model_cards/VibrantForests/VibrantForests.md

```diff
@@ -0,0 +1,264 @@
+* **AGB**: aboveground live tree biomass [Mg/ha]
+* **QMD**: quadratic mean diameter [cm]
+* **BA**: basal area of live trees [m²/ha]
+* **Cover**: canopy cover [%]
+* **Height**: canopy height [cm]
+
+The additional metrics trees per hectare (**TPH**) and Stand Density Index (**SDI**) are then derived deterministically.
+
+VibrantForests was trained using a context window of 256 x 256 pixels, enabling the model to capture spatial patterns from local- to stand-scales. Inference was executed using sliding overlapping windows, which were subsequently mosaicked together to produce seamless raster outputs.
+
+### Contributors
+Luke Zachmann, David Diaz, Tony Chang, Nathan Rutenbeck, Vincent Landau, Katharyn Duffy, Andreas Gros, Kiarie Ndegwa, Scott Conway, Guy Bayes
+
+### Model date
+November 2025
+
+### Model version
+1.1.1
+
+### Model type
+VibrantForests uses a Feature Pyramid Network (FPN) architecture built on VibrantMAE, a Vision Transformer-based Masked AutoEncoder (MAE). The VibrantMAE encoder serves as a frozen feature extractor, producing multi-scale feature representations extracted at multiple transformer depths. These features are then processed through the FPN decoder and a pixelwise regression head to predict the five forest structure attributes.
+
+## Intended use
+
+### Primary intended uses
+This model was designed to support forest mapping and monitoring with direct applications in a decision support system for wildfire risk assessment and land management planning. It is intended to produce estimates with annual update frequency at sufficient resolution to enable planning and prioritization activities at spatial scales ranging from individual forest stands up to large landscapes. The model was developed with an initial geographic focus on the contiguous United States (CONUS) following modeling approaches that can be readily extended to other forested regions. 
+
+VibrantForests was also developed to provide a springboard from which additional forest attributes could be derived (e.g., relative density, forest product volumes, etc.). Together, these raw and derived attributes are intended to enable assessments of forest management impacts on the land including evaluations of the efficacy of management interventions at landscape scales at achieving objectives like reducing the extent and severity of wildfires and associated risks to communities.
+
+### Primary intended users
+The intended direct users of VibrantForests outputs are natural resource managers and other professionals leading wildfire risk assessments, community wildfire protection planning, and forest restoration planning through the [Vibrant Planet Platform](https://www.vibrantplanet.net/platform). These use cases generally involve access to zonal summaries of forest and other land attributes at the scale of management units rather than accessing raster data directly. 
```

Comment:
We should link to paper when it becomes available. There is a preprint here: https://osf.io/7mcxq/overview